packages/sanes-chrome-extension/src/routes/login/index.dom.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { Store } from 'redux';
+import { RootState } from '../../store/reducers';
+import { aNewStore } from '../../store';
+import TestUtils from 'react-dom/test-utils';
+import { travelToLogin } from './test/travelToLogin';
+import { whenOnNavigatedToRoute } from '../../utils/test/navigation';
+import { WELCOME_ROUTE } from '../paths';
+
+describe('DOM > Feature > Login', () => {
+  let store: Store<RootState>;
+
+  beforeEach(() => {
+    store = aNewStore();
+  });
+
+  it(`should two inputs and one button`, async () => {
+    const LoginDom = await travelToLogin(store);
+
+    const inputs = TestUtils.scryRenderedDOMComponentsWithTag(
+      LoginDom,
+      'input'
+    );
+
+    expect(inputs.length).toBe(2);
+
+    TestUtils.findRenderedDOMComponentWithTag(LoginDom, 'button');
```

Comment:
why this line?

Reply:
There should be only one button on this view.

Reply:
But I will update this line for better readability